tests/std/tests/GH_004657_expected_constraints_permissive/test.compile.pass.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <expected>
+// <locale> provides several explicit template instantiation definitions that might break constraints checking
+// and hence cause errors in /permissive modes.
+#include <locale>

```

Comment:
I restored the inclusion of `<locale>`. The detailed reason can be found in the discussion in #4657.